docker-compose.yml

```diff
@@ -54,3 +69,4 @@ networks:
             com.docker.network.driver.mtu: ${DOCKER_MTU:-1500}
     backend:
         driver: overlay
+        # external: true # uncomment if you want to re-use and existing network! if not it will create a network dedicated to the stack!
```

Comment:
Since this file is used in the recipes, it may be better to keep it external and add the network create and network remove in the test script. @chicco785 wdyt?